api/src/test/java/org/openmrs/util/OpenmrsUtilTest.java

```diff
@@ -755,4 +761,172 @@ public void copyFile_shouldCopyInputstreamToOutputstreamAndCloseTheOutputstream(
 		verify(output, times(1)).close();
 	}
 
+	/**
+	 * @see OpenmrsUtil#isTestMode()
+	 */
+	@Test
+	public void isTestMode_shouldPassIfFunctionalTestModeIsEnabled() {
+		
+		System.setProperty("FUNCTIONAL_TEST_MODE", Boolean.toString(true));
+		boolean isTestMode = OpenmrsUtil.isTestMode();
+
+		assertTrue(isTestMode);
+	}
+
+	/**
+	 * @see OpenmrsUtil#isTestMode()
+	 */
+	@Test
+	public void isTestMode_shouldPassIfFunctionalTestModeIsDisabled() {
```

Comment:
rename test method to `isTestMode_shouldReturnFalseIfFunctionalTestModeIsDisabled`